### DISCUSSION TITLE
GEF Website?

### DISCUSSION THREAD

**Discussion by ptziegler** [eclipse-gef/gef-classic#411] [General]
Our readme is referencing the following webpages, but when clicking on them, I get redirected to the [project page](https://projects.eclipse.org/projects/tools.gef):

https://www.eclipse.org/gef/gef_mvc/index.php
https://www.eclipse.org/gef/draw2d/index.php
https://www.eclipse.org/gef/zest/index.php

Does anyone know what happened to them? The GEF website as a whole seems to have been available up until early [2023](https://web.archive.org/web/20230324030505/https://www.eclipse.org/gef/) but even then, GEF Classic is no longer mentioned.
Those entries seem to have been removed sometime during [2016](https://web.archive.org/web/20160531203516/https://www.eclipse.org/gef) when the main focus was moved to FX.

Does GEF Classic even need a website? Otherwise I would take those old pages, convert them to Markdown and add them to the GitHub wiki.

**Comment by azoitl** [comment]
The pages should still be in the git repo of the GEF web-page. I have lost the link on the new laptop. I'll try to look on my old machine in the next days. When I started GEF Classic I didn't want to interfere to much with the GEF website. And mostly used the wiki as a kind of web-page replacement for GEF Classic.

**Reply by ptziegler** [reply]
> The pages should still be in the git repo of the GEF web-page. 

You're right... The pages are still there:
https://git.eclipse.org/c/www.eclipse.org/gef.git/tree/

It also links to a corresponding helpdesk item, which has been left unresolved:
https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/1796

With the end of php already at the horizon, we wouldn't be able to simply revive that project and I already had my fill of Hugo so I'm not overly keen on migrating such a big website...

**Reply by ptziegler** [reply]
From my perspective, I think the best solution is to salvage what's worth keeping and putting it to the wiki. That also has the added benefit that everything is in a single repository as opposed to being scattered everywhere.
Which I already did for the GEF, Draw2D and Zest page, btw.
